core/templates/components/material.module.ts

```diff
@@ -22,10 +22,13 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatCardModule } from '@angular/material/card';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
+import { MatSelectModule } from '@angular/material/select';
```

Comment:
Please remove the line break above this line. Also, please sort these angular imports according to the import path.

Reply:
Done